frontend/apps/web/src/pages/index.tsx

```diff
@@ -1,152 +1,156 @@
-import { RepositoryData } from '@/mocks'
+import { PageContainer } from '@/containers/PageContainer'
 import {
   Button,
-  Icon,
-  NativeLink,
+  Link,
   PageLayout,
   ProgressBar,
   StackLayout,
   Table,
   Text,
   usePageLayout,
 } from '@filecoin/ui'
-import React, { useState } from 'react'
-import { NavLink, useNavigate } from 'react-router-dom'
+import { styled } from '@storybook/theming'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { getButton } from './all-tests'
 
 const Header = props => {
-  const [activeTab, setActiveTab] = useState(1)
-
+  const { t } = useTranslation()
+  const navigate = useNavigate()
   return (
     <PageLayout.Header>
-      <StackLayout>
-        <Text type="heading 5">Systems</Text>
-      </StackLayout>
-      <PageLayout.Tabs>
-        <NavLink to={'/system/overview'}>
-          <PageLayout.Tab
-            onClick={() => setActiveTab(1)}
-            active={activeTab === 1}
+      <HeaderWrapper>
```

Comment:
I think, this could be accomplished with ColumnLayout and split property, but, I'm not sure.

Reply:
There is a part of code in ColumnLayout on line 139:
```
return `
          > *:nth-child(${split}) {
            margin-right: auto;
            
            + * {
                margin-left: 0;
            }
          }`
```

Reply:
So basicly if you have 4 elements in ColumnLayout and a split={1} it should put margin-right: auto on the first element and that should push all others to the right. Let me know if it works

Reply:
![image](https://user-images.githubusercontent.com/85948457/143460072-3958c96e-61e3-4a5d-a3ce-ecc82f824ebf.png)
It isn't working